src/_includes/layouts/projects.njk

```diff
@@ -0,0 +1,20 @@
+{% extends "layouts/base.njk" %}
+
+{% block content %}
+<div>
+    <section>
```

Comment:
Same as my comment above— better to actually build this page so it starts working once we have actual content than to include non-functional boilerplate.